### PR TITLE
🎉Source Stripe: New Stream - Early Fraud Warnings

### LINKED ISSUE #16962: Source Stripe: new stream Early Fraud Warning
## Tell us about the problem you're trying to solve
[Early Fraud Warning](https://stripe.com/docs/api/radar/early_fraud_warnings)
An early fraud warning indicates that the card issuer has notified us that a charge may be fraudulent..

[Permission](https://stripe.com/docs/security/permissions)

## Describe the solution you’d like
Research pros and cons for implementing this stream.
_Research should contain: what stream should be implemented, permissions and scopes (if needed), paid plan (if needed)._

### PR DESCRIPTION
**Pull request by btkcodedev**
## What
Closes #16962 

## Enhancement
New streams: 

- Early fraud warnings 

## Recommended reading order
1. `source_stripe/source.py`
2. `source_stripe/streams.py`
3.  `source_stripe/schemas/early_fraud_warnings.json`

## 🚨 User Impact 🚨
No breaking changes

## Pre-merge Checklist
<details open><summary><strong>Updating a connector</strong></summary>

### Community member or Airbyter [Yes]

- [x] Grant edit access to maintainers ([instructions](https://docs.github.com/en/github/collaborating-with-pull-requests/working-with-forks/allowing-changes-to-a-pull-request-branch-created-from-a-fork#enabling-repository-maintainer-permissions-on-existing-pull-requests))
- [x] Secrets in the connector's spec are annotated with `airbyte_secret`
- [x] Unit & integration tests added and passing. Community members, please provide proof of success locally e.g: screenshot or copy-paste unit, integration, and acceptance test output. To run acceptance tests for a Python connector, follow instructions in the README. For java connectors run `./gradlew :airbyte-integrations:connectors:<name>:integrationTest`.
- [x] Issue acceptance criteria met
- [x] PR name follows [PR naming conventions](https://docs.airbyte.com/contributing-to-airbyte/issues-and-pull-requests)
- [x] Documentation which references the generator is updated as needed

## For the reviewer: 
Enhancement passing in local python integration tests. Needs only to test deployment to cloud build and publish it :)

### CODE REVIEW

**Comment by YowanR** [comment]
@sh4sh @natalyjazzviolin can you please take a look? 🙏

**Comment by evantahler** [comment]
@natalyjazzviolin those github checks are failing because I think we need a branch that's really in our repo (not a fork)
>  Create a non-forked branch based on this PR and test the below items on it

**Comment by btkcodedev** [comment]
@natalyjazzviolin Could I help for anything?

**Comment by btkcodedev** [comment]
@natalyjazzviolin @evantahler Little Bump Here :)

**Comment by natalyjazzviolin** [comment]
@evantahler sorry for the confusion - my understanding was that a review is required by the connector Ops team because this is a GA connector. I've created a non-forked branch to run the build, it's still running as I type this so don't know if it's passed.

**Comment by evantahler** [comment]
Testing and deploying from https://github.com/airbytehq/airbyte/pull/24062

**Comment by evantahler** [comment]
Closing in favor of https://github.com/airbytehq/airbyte/pull/24062

**Comment by btkcodedev** [comment]
@sajarin All comments resolved, Docs and version updated.
Please forward with the review :)

**Comment by btkcodedev** [comment]
Secrets are available from lastpass

**Comment by btkcodedev** [comment]
I can't seed the stream as it doesn't support POST operation, however added it as empty stream.

File: airbyte-config-oss/init-oss/src/main/resources/seed/source_definitions.yaml [hunk 1]
```diff
@@ -2041,7 +2041,7 @@
 - name: Stripe
   sourceDefinitionId: e094cb9a-26de-4645-8761-65c0c425d1de
   dockerRepository: airbyte/source-stripe
-  dockerImageTag: 3.4.2
+  dockerImageTag: 3.5.2
```
Comment (erohmensing) [inline airbyte-config-oss/init-oss/src/main/resources/seed/source_definitions.yaml:2044]:
```suggestion
  dockerImageTag: 3.5.0
```
When bumping minor versions, please bump back to .0 :)

File: docs/integrations/sources/stripe.md [hunk 2]
```diff
@@ -85,13 +86,14 @@ The Stripe connector should not run into Stripe API limitations under normal usa
...
 |:--------|:-----------|:---------------------------------------------------------|:-----------------------------------------------------------------------------------------------------------------------------------------------------|
-| 3.4.2   | 2023-05-04 | [25795](https://github.com/airbytehq/airbyte/pull/25795) | Added `CDK TypeTransformer` to guarantee declared JSON Schema data-types  |
-| 3.4.1   | 2023-04-24 | [23389](https://github.com/airbytehq/airbyte/pull/23389) | Add `customer_tax_ids` to `Invoices` |
-| 3.4.0   | 2023-03-20 | [23963](https://github.com/airbytehq/airbyte/pull/23963) | Add `SetupIntents` stream |
-| 3.3.0   | 2023-04-12 | [25136](https://github.com/airbytehq/airbyte/pull/25136) | Add stream `Accounts`                                                        |
+| 3.5.2   | 2023-05-10 | [22859](https://github.com/airbytehq/airbyte/pull/22859) | Added stream `Early Fraud Warnings`                                                                                                                  |
```
Comment (erohmensing) [inline docs/integrations/sources/stripe.md:89]:
```suggestion
| 3.5.0   | 2023-05-10 | [22859](https://github.com/airbytehq/airbyte/pull/22859) | Added stream `Early Fraud Warnings`                                                                                                                  |
```

File: airbyte-integrations/connectors/source-stripe/acceptance-test-config.yml [hunk 3]
```diff
@@ -22,6 +22,8 @@ acceptance_tests:
         empty_streams:
           - name: "bank_accounts"
             bypass_reason: "This stream can't be seeded in our sandbox account"
+          - name: "early_fraud_warnings"
+            bypass_reason: "This stream can't be seeded in our sandbox account"
```
Comment (erohmensing) [inline airbyte-integrations/connectors/source-stripe/acceptance-test-config.yml:26]: Bummer, but I can see how this makes sense

**Review by erohmensing — APPROVED** [review]
Seems simple enough! Made a suggestion around correct version bumping. Good from the connector ops side once the version is updated. Please also fill out the checklist from the checklist action :)

File: airbyte-integrations/connectors/source-stripe/Dockerfile [hunk 4]
```diff
@@ -14,5 +14,5 @@ ENTRYPOINT ["python", "/airbyte/integration_code/main.py"]
 
 
 
-LABEL io.airbyte.version=3.4.2
+LABEL io.airbyte.version=3.5.2
```
Comment (alafanechere) [inline airbyte-integrations/connectors/source-stripe/Dockerfile:17]:
```suggestion
LABEL io.airbyte.version=3.5.0
```
And please update the `metadata.yaml` file with the same version number 😄

**Comment by sajarin** [comment]
@btkcodedev can you rebase on master and address the comments above?

**Comment by btkcodedev** [comment]
@sajarin All comments resolved, Conflicts resolved & Version bumped.
Thanks :)

**Comment by marcosmarxm** [comment]
Waiting https://github.com/airbytehq/airbyte/pull/26360 to fix the CI for Stripe.

**Review by marcosmarxm — APPROVED** [review]

**Pull request merged**